packages/terra-data-grid/src/WorklistDataGrid.module.scss

```diff
@@ -1,29 +1,7 @@
-/* stylelint-disable terra/custom-property-namespace */
-@import '~terra-mixins/lib/Mixins';
-
-// Themes
-@import './orion-fusion-theme/WorklistDataGrid.module';
-@import './clinical-lowlight-theme/WorklistDataGrid.module';
-
 :local {
-  .worklist-data-grid :focus {
-    outline: var(--terra-worklist-data-grid-cell-focus-outline, 3px dashed #000);
-    outline-offset: var(--terra-worklist-data-grid-cell-focus-outline-offset, -2px);
-  }
-
   .worklist-data-grid-container {
     height: 100%;
     overflow: auto;
```

Comment:
Are you just creating another container for event listeners?  I don't think the WorklistDataGrid needs an overflow both vertically and horizontally.  That is being handled by the DataGrid component.  You would be introducing the possibility of multiple scrollbars.